packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/modules/DevMenuInternalModule.kt

```diff
@@ -25,6 +26,15 @@ class DevMenuInternalModule(reactContext: ReactApplicationContext)
       .getNativeModule(DevMenuSettings::class.java)
   }
 
+  private val isDeviceSupportsKeyCommands
```

Comment:
`is device supports` is incorrect in English, my suggestions:

`doesDeviceSupportKeyCommands`, `isDeviceSupportingKeyCommands`, `deviceSupportsKeyCommands`

However the second suggests that this value is valid only at the time of asking and may change later, so I'd choose the first or third as they're more general.